client/src/App.vue

```diff
@@ -247,6 +247,10 @@ export default class App extends Mixins(SessionTimerMixin) {
   background-color: rgb(33, 33, 33) // grey darken-4
   border-color: rgb(33, 33, 33) // grey darken-4
 
+.app-header
+  .v-btn
+    box-shadow: unset !important
```

Comment:
This is the simplest solution but I'm open to other ideas.

The problem is that the header uses different styles than the rest of the app. But here, our app styles were affecting the header buttons (eg, Login).

Or, we don't worry about it for now.

Reply:
@cameron-freshworks Haha, I see you did (almost) the same thing in your recent PR.

Reply:
hahah i did! 
No, i just happened to notice it after rebasing :)